RecoTracker/IterativeTracking/python/JetCoreRegionalStep_cff.py

```diff
@@ -63,35 +63,38 @@
     ]
 )
 
-# SEEDS
-import RecoTracker.TkSeedGenerator.GlobalSeedsFromPairsWithVertices_cff
-jetCoreRegionalStepSeeds = RecoTracker.TkSeedGenerator.GlobalSeedsFromPairsWithVertices_cff.globalSeedsFromPairsWithVertices.clone()
-jetCoreRegionalStepSeeds.RegionFactoryPSet = cms.PSet(
-      ComponentName = cms.string( "TauRegionalPixelSeedGenerator" ),#not so nice to depend on RecoTau...
-      RegionPSet = cms.PSet(
-        precise = cms.bool( True ),
-        originRadius = cms.double( 0.2 ),
-        ptMin = cms.double( 10. ),
-        originHalfLength = cms.double( 0.2 ),
-        deltaPhiRegion = cms.double( 0.20 ), 
-        deltaEtaRegion = cms.double( 0.20 ), 
-        JetSrc = cms.InputTag( "jetsForCoreTracking" ),
-#       JetSrc = cms.InputTag( "ak5CaloJets" ),
-        vertexSrc = cms.InputTag( "firstStepGoodPrimaryVertices" ),
-        measurementTrackerName = cms.string( "MeasurementTrackerEvent" ),
-        howToUseMeasurementTracker = cms.string( "Never" )
-      )
+# TrackingRegion
+from RecoTauTag.HLTProducers.tauRegionalPixelSeedTrackingRegions_cfi import tauRegionalPixelSeedTrackingRegions as _tauRegionalPixelSeedTrackingRegions
+jetCoreRegionalStepTrackingRegions = _tauRegionalPixelSeedTrackingRegions.clone(RegionPSet=dict(
+    ptMin = 10,
+    deltaPhiRegion = 0.20,
+    deltaEtaRegion = 0.20,
+    JetSrc = "jetsForCoreTracking",
+#    JetSrc = "ak5CaloJets",
+    vertexSrc = "firstStepGoodPrimaryVertices",
+    howToUseMeasurementTracker = "Never"
+))
+
+# Seeding
+from RecoTracker.TkHitPairs.hitPairEDProducer_cff import hitPairEDProducer as _hitPairEDProducer
+jetCoreRegionalStepHitDoublets = _hitPairEDProducer.clone(
+    seedingLayers = "jetCoreRegionalStepSeedLayers",
+    trackingRegions = "jetCoreRegionalStepTrackingRegions",
+    produceSeedingHitSets = True,
 )
-jetCoreRegionalStepSeeds.OrderedHitsFactoryPSet.SeedingLayers = 'jetCoreRegionalStepSeedLayers'
-jetCoreRegionalStepSeeds.SeedComparitorPSet = cms.PSet(
-        ComponentName = cms.string('none'),
-#PixelClusterShapeSeedComparitor'),
+from RecoTracker.TkSeedGenerator.seedCreatorFromRegionConsecutiveHitsEDProducer_cff import seedCreatorFromRegionConsecutiveHitsEDProducer as _seedCreatorFromRegionConsecutiveHitsEDProducer
+jetCoreRegionalStepSeeds = _seedCreatorFromRegionConsecutiveHitsEDProducer.clone(
+    seedingHitSets = "jetCoreRegionalStepHitDoublets",
+    forceKinematicWithRegionDirection = True
+#    SeedComparitorPSet = dict(# FIXME: is this defined in any cfi that could be imported instead of copy-paste?
+#        ComponentName = 'PixelClusterShapeSeedComparitor',
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant